src/components/SearchBar.jsx

```diff
@@ -1 +1,31 @@
 // implement SearchBar component here
+import React from 'react';
+
+export default class SearchBar extends React.Component {
+  render() {
+    const { onSearchTextChange, onBookmarkedChange, onSelectedGenreChange } = this.props;
+    return (
+      <div>
+        <form>
+          <label htmlFor="search">Inclui o texto:</label>
+          <input
+            type="text" name="search" id="search" value={this.props.searchText}
```

Comment:
Aí aqui pode tirar o this.props :)
```suggestion
            type="text" name="search" id="search" value={searchText}
```